client/src/modules/inventory/list/templates/action.cell.html

```diff
@@ -8,26 +8,28 @@
   <ul class="dropdown-menu-right" bh-dropdown-menu-auto-dropup uib-dropdown-menu>
     <li>
       <a href
-          data-method="edit"
-          data-edit-metadata="{{ row.entity.code }}"
-          ng-hide="row.groupHeader" ui-sref="inventory.update({uuid : row.entity.uuid })">
-          <i class="fa fa-edit"></i> <span translate>FORM.BUTTONS.EDIT</span>
+        data-method="edit"
+        data-edit-metadata="{{ row.entity.code }}"
+        ui-sref="inventory.update({uuid : row.entity.uuid })">
+        <i class="fa fa-edit"></i> <span translate>FORM.BUTTONS.EDIT</span>
       </a>
     </li>
+    <li class="divider"></li>
```

Comment:
Having a `divider` here is for separating concepts ?

Reply:
Yup!  It also adds more space so that we don't accidentally click it when we try to click "edit".